src/setup-wp-server-files.ts

```diff
@@ -48,9 +49,28 @@ async function copyBundledSqlite() {
 	await recursiveCopyDirectory( bundledSqlitePath, getSqlitePath() );
 }
 
+async function copyBundledWPCLI() {
+	const bundledWPCLIInstalled = await fs.pathExists( getWpCliPath() );
+	console.log( `Copying bundled WP-CLI...` );
+	console.log( bundledWPCLIInstalled );
+	if ( bundledWPCLIInstalled ) {
+		return;
+	}
+	const bundledWPCLIPath = path.join( getResourcesPath(), 'wp-files', 'wp-cli', 'wp-cli.phar' );
+	await fs.copyFile( bundledWPCLIPath, getWpCliPath() );
+}
+
+async function updateLatestWPCLI() {
+	const shouldOverwrite = true;
+	const res = await downloadWPCLI( shouldOverwrite );
+	console.log( `WP-CLI downloaded:`, res );
+}
```

Comment:
Let's remove updateLatestWPCLI and handle it in a different issue https://github.com/Automattic/dotcom-forge/issues/7326

I would expect that it compares the current version with the latest release and download it only if necessary

Reply:
Yes, I agree, this is a better approach! Let's do that!